Documentation/Usage.md

```diff
@@ -397,3 +398,34 @@ manifest API).
 It is *not* expected the packages would ever use this feature unless absolutely
 necessary to support existing clients. In particular, packages *should not*
 adopt this syntax for tagging versions supporting the _latest GM_ Swift version.
+
+### Editable Packages
+
+SwiftPM supports editing dependencies, which means moving it into a location under `Packages/` directory. 
+
+If a such an editable package is present in Packages, then swift build will always use the exact sources in this directory to build, regardless of it's state, git repository status, tags, or the tag desired by dependency resolution. In other words, this will "just build" against the sources that are present.
```

Comment:
Rather than say "present in Packages", I would talk about "packages which are in the editable state".